.github/workflows/release-homebrew.yaml

```diff
@@ -14,5 +14,5 @@ jobs:
         tap: microsoft/git
         name: git-credential-manager-core
         type: cask
-        releaseAsset: gcmcore-osx-(.*)\.pkg
+        releaseAsset: gcm-osx-(.*)\.pkg
```

Comment:
This change belongs in the third commit, not the first, correct?

Reply:
Yep! Rebase fail